src/CMakeLists.txt

```diff
@@ -340,6 +340,8 @@ libcvc5_add_sources(
   smt/proof_final_callback.h
   smt/proof_post_processor.cpp
   smt/proof_post_processor.h
+  smt/proof_post_processor_dsl.cpp
+  smt/proof_post_processor_dsl.h
```

Comment:
maybe we wanna start changing "DSL" in general to "RARE"?

Reply:
Yes I agree.

I'll do a pass separately to change this. I'd prefer not changing now to avoid conflicts in current dev branches.